generators/client/templates/src/components/layout/TopBar.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import SignOutButton from "../authentication/SignOutButton";
+
+const TopBar = ({ user }) => {
+  const unauthenticatedListItems = [
+    <li key="sign-in">
+      <Link to="/user-sessions/new">Sign In</Link>
+    </li>,
+    <li key="sign-up">
+      <Link to="/users/new" className="button">
+        Sign Up
+      </Link>
+    </li>,
+  ];
+
+  const authenticatedListItems = [
+    <li key="sign-out">
+      <SignOutButton />
+    </li>,
+  ];
+
+  return (
+    <div className="top-bar">
+      <div className="top-bar-left">
+        <ul className="menu">
+          <li className="menu-text">App</li>
+          <li>
+            <Link to="/home">Home</Link>
```

Comment:
can we instead have this point toward the root route? Otherwise, clicking and then refreshing causes a 404 error.